config.json

```diff
@@ -2,9 +2,9 @@
 	"database": {
 		"host": "localhost",
 		"port": "5432",
-		"user": "gorm",
-		"password": "gorm",
-		"dbname": "gorm"
+		"user": "postgres",
+		"password": "postgres",
+		"dbname": "shadowguard"
```

Comment:
You must update the build scripts to reflect these changes so the database is built correctly.

Reply:
Fixed!

Reply:
Perfect, looks good to me